test/browser-tests/twoway.js

```diff
@@ -603,7 +603,11 @@ test( 'Changes made in oninit are reflected on render (#1390)', t => {
 	new Ractive({
 		el: fixture,
 		template: '{{#each items}}<input type="checkbox" name="{{selected}}" value="{{this}}">{{/each}}',
-		data: { items: [ 'a', 'b', 'c' ] },
+		data: {
+			items: [ 'a', 'b', 'c' ],
+			// will fail without this
+			selected: null
+		},
```

Comment:
Agree, but that would have sent me into some serious yak shaving.

Two things happened: 1) the new code immediately forces binding to happen AND 2) forced resolution now resolves to context scope, not instance scope (a good thing). 

Except that for multi-select twoways it will end up trying to bind to the first instance in the loop when probably some higher context was intended.

FWIW, it does warn about the ambiguity. 

Ideally I think bindings would wait till the last responsible moment to force bind, then it would work as long as set happened before some other action like user clicking on a checkbox or a checkbox having a `checked` attribute.

In any case, I added an additional test that demonstrated the issue outside of change to `oninit` to show it wasn't strictly related to this change - and hence wanted to divorce the two things and avoid:

![image](https://cloud.githubusercontent.com/assets/478864/10077370/e5d867b0-6294-11e5-9bfd-002e5d559c62.png)